build/signedbuild.yml

```diff
@@ -17,347 +17,351 @@ variables:
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: NuGet package'
-    inputs:
-      PathtoPublish: 'src\CI\bin\Release\NuGet'
-      ArtifactName: 'NuGet'
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: Rules markdown'
-    inputs:
-      PathtoPublish: 'src\RulesMD\bin\Release\axe-windows-rules.md'
-      ArtifactName: 'axe-windows-rules'
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: CLI (msi)'
-    inputs:
-      PathtoPublish: 'src\CLI_Installer\bin\x86\Release\AxeWindowsCLI.msi'
-      ArtifactName: 'CLI-msi'
-
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: CLI (zip)'
-    inputs:
-      PathtoPublish: 'src\CLI_Installer\bin\x86\Release\AxeWindowsCLI.zip'
-      ArtifactName: 'CLI-zip'
-
-  - task: CopyFiles@2
-    displayName: 'Copy CLI_Full Files for Signing Validation'
-    inputs:
-      SourceFolder: '$(Build.SourcesDirectory)\src'
-      Contents: |
-       CLI_Full\bin\release\net6.0\win7-x86\?(*.exe|*.dll)
-      TargetFolder: '$(Build.ArtifactStagingDirectory)\SigningValidation'
-
-  - task: CopyFiles@2
-    displayName: 'Copy CLI Files for Signing Validation'
-    inputs:
-      SourceFolder: '$(Build.SourcesDirectory)\src'
-      Contents: 'CLI\bin\release\net6.0\**\?(*.exe|*.dll)'
-      TargetFolder: '$(Build.ArtifactStagingDirectory)\SigningValidation'
-
-  - task: ms-vseng.MicroBuildTasks.521a94ea-9e68-468a-8167-6dcf361ea776.MicroBuildCleanup@1
-    displayName: 'Perform Cleanup Tasks'
-    condition: succeededOrFailed()
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    pool: 
+      name: MSEngSS-MicroBuild2022-1ES
+    sdl:
+      binskim:
+        analyzeTargetGlob: "$(System.DefaultWorkingDirectory)\\src\\CI\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Actions\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Automation\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Core\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Desktop\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Rules\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\RuleSelection\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\SystemAbstractions\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Telemetry\\bin\\**\\*.dll;\
+                            $(System.DefaultWorkingDirectory)\\src\\Win32\\bin\\**\\*.dll;"
+
+                
+    stages:
+    - stage: Stage
+      jobs:
+      - job: ComplianceRelease
```

Comment:
General comment: The old file used newlines to try to break it up into chunks. the new file has removed the newlines, which makes it a lot harder to identify the various blocks. Can we please restore newlines between pieces?

Reply:
reverted the new lines